Data-Pro-Test/Controllers/RecipesController.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using System.Net.Mail;
+using System.Net;
+
+namespace Data_Pro_Test.Controllers
+{
+    public class RecipesController : Controller
+    {
+        // GET: Recipes
+        public ActionResult Index()
+        {
+            return View();
+        }
+
+        // GET: Recipes/Details/5
+        public ActionResult Details(int id)
+        {
+            return View();
+        }
+
+        // GET: Recipes/Create
+        public ActionResult Create()
+        {
+            return View();
+        }
+
+        // POST: Recipes/Create
+        [HttpPost]
+        public ActionResult Create(FormCollection collection)
+        {
+            try
+            {
+                // TODO: Add insert logic here
+
+                return RedirectToAction("Index");
+            }
+            catch
+            {
+                return View();
+            }
+        }
+
+        // GET: Recipes/Edit/5
+        public ActionResult Edit(int id)
+        {
+            return View();
+        }
+
+        // POST: Recipes/Edit/5
+        [HttpPost]
+        public ActionResult Edit(int id, FormCollection collection)
+        {
+            try
+            {
+                // TODO: Add update logic here
+
+                return RedirectToAction("Index");
+            }
+            catch
+            {
+                return View();
+            }
+        }
+
+        // GET: Recipes/Delete/5
+        public ActionResult Delete(int id)
+        {
+            return View();
+        }
+
+        // POST: Recipes/Delete/5
+        [HttpPost]
+        public ActionResult Delete(int id, FormCollection collection)
+        {
+            try
+            {
+                // TODO: Add delete logic here
+
+                return RedirectToAction("Index");
+            }
+            catch
+            {
+                return View();
+            }
+        }
+
+        [HttpPost]
+        public ActionResult sendEmail(string name, string phone, string email, string city, string getPost, string friend)
```

Comment:
Move all params to be field of vie model.
So the action will get one object with all the params inside.